rohmu/object_storage/azure.py

```diff
@@ -67,13 +71,13 @@ class AzureTransfer(BaseTransfer[Config]):
 
     def __init__(
         self,
-        bucket_name,
-        account_name,
-        account_key=None,
-        sas_token=None,
-        prefix=None,
-        azure_cloud=None,
-        proxy_info=None,
+        bucket_name: str,
```

Comment:
It seems that we need named parameters in this constructor, because most of them are optional

Reply:
Public API -> please no signature change unless really needed.